lib.rs

```diff
@@ -38,29 +56,26 @@ mod token {
     #[ink(event)]
     pub struct Transfer {
         #[ink(topic)]
-        from: AccountId,
+        from: Option<AccountId>,
         #[ink(topic)]
-        to: AccountId,
+        to: Option<AccountId>,
```

Comment:
when `to` can be `None`?

Reply:
Or `from`. It doesn't seem to be representable in the API of PSP22 standard - i.e. you can't transfer from/to `None`.

Reply:
It's reserved for handling minting and burning tokens. I'm just trying to follow the standard defined here:
https://github.com/w3f/PSPs/blob/master/PSPs/psp-22.md#transfer